### ISSUE TITLE
FC4-01 | Inconsistent Token Symbols

### ISSUE THREAD

**Issue by Brando753** [Sifchain/sifnode#3038]
This issue appears to have been closed with issue #2768 but should still be verified in the current base as it is labeled critical.
Verify that the following files either no longer check for or are consistent in their usage of EVMNATIVE to close this ticket out:

- [ ] BridgeBank.sol
- [ ] parser.go
- [ ] msgs.go

# Description
When the user locks EVM native token to the `BridgeBank` contract, the symbol in the event submitted by the `handleNativeCurrencyLock()` function is `EVMNATIVE`.

```solidity
function handleNativeCurrencyLock(bytes calldata recipient, uint256 amount)
    internal
    validSifAddress(recipient)
    onlyNotBlocklisted(msg.sender)
  {
    require(msg.value == amount, "amount mismatch");
    address token = address(0);
    lockBurnNonce = lockBurnNonce + 1;
    emit LogLock(
      msg.sender,
      recipient,
      token,
      amount,
      lockBurnNonce,
      18, // decimals
      "EVMNATIVE", // symbol
      "EVMNATIVE", // name
      networkDescriptor
    );
  }
```

However, in the file `cmd/ebrelayer/txs/parser.go`, when the relayer parses the events of the EVM chain, the symbol used to determine whether the locked asset is EVM native tokens is `eth`.
```go
symbol := strings.ToLower(event.Symbol)
	switch event.ClaimType {
	case ethbridge.ClaimType_CLAIM_TYPE_LOCK:
		if symbol == "eth" && !isZeroAddress(event.Token) { 
			return ethBridgeClaim, 
                 errors.New("symbol \"eth\" must have null address set as token address")
		}
	case ethbridge.ClaimType_CLAIM_TYPE_BURN:
		symbol = symbolTranslator.EthereumToSifchain(symbol)
	}
```
In the file `x/ethbridge/types/msgs.go`, the symbol of native asset in EVM chain is `eth` too.
```go
func (msg MsgCreateEthBridgeClaim) ValidateBasic() error {
	...
	if strings.ToLower(msg.EthBridgeClaim.Symbol) == "eth" &&
		NewEthereumAddress(msg.EthBridgeClaim.TokenContractAddress) != 
  NewEthereumAddress("0x0000000000000000000000000000000000000000") {
		return ErrInvalidEthSymbol
	}
    ...
}
```
Inconsistent symbols might lead to cross-chain transaction failure.

# Recomendation
Recommend keeping the same symbol for the Ethereum native token on both chains.

**Comment by juniuszhou** [comment]
Now, we use the denom that combine network id and contract address to identify the token. native eth can be recognized by its contract as all zero. some code already removed in future/peggy2 branch.

**Comment by banshee** [comment]
Agree that this has been addressed by 32fd2f5444fb380f2181e366f179f592c43480e4 and a3c651ca7015640225a65b973258be7d91d6ecff.  Ok to close this issue.

**Comment by smartyalgo** [comment]
Agreed, no longer an issue,
